easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.11-foss-2020b.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2020.11'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('pybind11', '2.6.0'),  # required by scipy
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# order is important!
+exts_list = [
+    ('hypothesis', '5.41.1', {
```

Comment:
We could consider moving this into Python?

Reply:
Since the Python 3.8.6 we're using in `2020b` is already included in the latest EasyBuild v4.3.1, that's relatively painful to do now (since people who already have Python 3.8.6 installed need to update it with `--skip --rebuild` or they'll run into `pip check` failures...).

Reply:
don't we do that all the time e.g. #11654  already?